### PR TITLE
Feature/win activity

### PR DESCRIPTION
**Pull request by josephosan**

### CODE REVIEW

File: app/src/main/java/aa/engine/jobs/critical/CollisionJob.kt [hunk 1]
```diff
@@ -44,7 +44,7 @@ class CollisionJob() : Job() {
                 && calDistance(
                     sBall.getPosition(),
                     closestBallToMainCircle.getPosition()
-                ) < smallBallCriticalDistance
+                ) <= smallBallCriticalDistance + 5
```
Comment (MahbodGhadiri) [inline app/src/main/java/aa/engine/jobs/critical/CollisionJob.kt:47]: whats the ```+5``` for?

File: app/src/main/java/aa/android/views/GameView.kt [hunk 2]
```diff
@@ -33,10 +34,13 @@ public class GameView(context: Context, attrs: AttributeSet) :
     private var reRenderReceiver: ReRenderReceiver;
     private var reRenderIntent: Intent;
     private var isFirstTime: Boolean = true;
+    private var activityContext: Context? = null
```
Comment (MahbodGhadiri) [inline app/src/main/java/aa/android/views/GameView.kt:37]: Doesn't ```this.context``` serve the same purpose?

**Review by MahbodGhadiri — CHANGES_REQUESTED** [review]

File: app/src/main/java/aa/engine/jobs/critical/CollisionJob.kt (hunk 1 above)
Comment (josephosan) [reply app/src/main/java/aa/engine/jobs/critical/CollisionJob.kt]: removed

File: app/src/main/java/aa/android/views/GameView.kt (hunk 2 above)
Comment (josephosan) [reply app/src/main/java/aa/android/views/GameView.kt]: managed

**Review by MahbodGhadiri — APPROVED** [review]
Well done!

**Pull request merged**